### DISCUSSION TITLE
Evaporation after removal dose

### DISCUSSION THREAD

**Discussion by MelanieMartha26** [Open-Systems-Pharmacology/Forum#1078] [Questions & Problems]
Dear everyone, 

I was trying to simulate the absorption of testosterone into the skin via the skin permeation model. I used a finite dose where I remove the dose after 8 hours. The vehicle type in this example is water (not volatile, not occluded). Surprisingly, after 8 hours, evaporation of the permeant occurs, which I find hard to believe. See picture below for the visualization. From the SC top layer evaporation equation in the passive transports building block (
![image](https://user-images.githubusercontent.com/97103784/149143243-d95283c0-bcc0-4e2b-9ce2-5b0efdb4958b.png)
), the evaporation should be 0 as the vehicle is not volatile and type 1. Does someone know what happens and whether it is reliable? 

The evaporation does not occur when ethanol is used as the vehicle (which is volatile).

![image](https://user-images.githubusercontent.com/97103784/149142635-b9588448-9a79-4e71-a054-dfac4d9a1bcd.png)

Thank you very much in advance!

Best regards, 

Mèlanie

**Comment by StephanSchaller** [comment]
Above formula should eval to 0 if vehicle_type = 1.

There must be sth. else then.
